docs/current/guides/757394-use-services.md

```diff
@@ -0,0 +1,455 @@
+---
+slug: /757394/use-services
+displayed_sidebar: "current"
+category: "guides"
+tags: ["go", "python", "nodejs"]
+authors: ["Alex Suraci"]
+date: "2023-10-12"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Embed from '@site/src/components/atoms/embed.js'
+
+# Use Services in Dagger
+
+:::warning
+Dagger v0.8.8 includes a breaking change for binding service containers to containers. The `Container.withServiceBinding` API now takes a `Service` instead of a `Container`, so you must call `Container.asService` on its argument. See the section on [binding container services to containers](#bind-service-containers-to-containers) for examples.
```

Comment:
non-blocking, but `v0.8.8` is out of date right? It appears again in this file a few times below

Reply:
good catch - bumped to v0.9.0 :+1: